stories/tree/__snapshots__/tree.stories.storyshot

```diff
@@ -1,959 +1,1104 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Storyshots Components/Tree Expanded Tree 1`] = `
+exports[`Storyshots Components/Tree RTL 1`] = `
+
+
+`;
+
+exports[`Storyshots Components/Tree Tree With Expanded Level 1 1`] = `
```

Comment:
I think it's best to simplify the titles: Tree Level 1. You have a description that explains what it does :)

Reply:
@jessmetivier you are looking at the `.storyshot` files. There is description for Level 1. You don't have to look at these `.stories.storyshot` files. You only need the `.stories.js` files for the documentation part.